### ISSUE TITLE
Please add Linux and MacOS support

### ISSUE THREAD

**Issue by ah1102** [moneriomaa/openear#19]
Please add Linux and MacOS support. Now many people use computers running these systems.

**Comment by moneriomaa** [comment]
all protocols in c++ language with no complicated library
but for GUI needs to spend some time for linux 
i do it but After the Windows version is complete
about why project is not open sourced some parts in projects is customized for corporation and i have no permit to share it. although rtl-usb is like security dongle for it 
i use this software for 5 years and decide to share with others,i think closed source of it better than never be it

**Comment by ah1102** [comment]
If you add Linux and macOS support, many will be happy. Will wait.

**Comment by moneriomaa** [comment]
I will definitely check this out

**Comment by st1905** [comment]
If macOS support is going to be added, maybe FreeBSD could also be included...

**Comment by ah1102** [comment]
Yes, and Linux is the same, since they are all in the Unix kernel.

**Comment by pyopower** [comment]
It would be very interesting to be able to enjoy it on Debian or Ubuntu
